package.json

```diff
@@ -34,16 +34,16 @@
     "yamljs": "^0.2.8"
   },
   "scripts": {
-    "postinstall": "npm run migrate:up",
+    "postinstall": "yarn migrate:up",
     "build": "webpack -p",
     "start": "node index.js",
     "debug": "node debug index.js",
     "watch": "NODE_ENV=development concurrently --kill-others --raw \"nodemon index.js --config nodemon.json\" \"webpack --progress --colors --watch\"",
-    "test": "npm run test:prepare && npm run test:server && npm run test:client",
```

Comment:
rather than updating all these "internal" uses of `npm run`, I thought about using something like https://github.com/camacho/yarn-or-npm to use whichever was available. It ended up feeling like unnecessary overhead so I opted to just change everything to plain ole `yarn`.